### PR TITLE
Add testing workflow and update contributing guide

### PR DESCRIPTION
**Pull request by codesue**
# What does this pull request do?

- Adds a basic workflow to run all unit tests on push or pull requests to `master` and release branches
- Updates contributing guide to include ways to contribute and instructions on contributing code
- Sets the min Python version to 3.7

Fixes https://github.com/tensorflow/model-card-toolkit/issues/239

## How did you test this change?

Successful workflow run on my branch: https://github.com/codesue/model-card-toolkit/actions/runs/3767521726

## How did you document this change?

Updated the contributing guide

## Before submitting

Before submitting a pull request, please be sure to do the following:
- [x] Read the [How to Contribute guide](https://github.com/tensorflow/model-card-toolkit/blob/master/CONTRIBUTING.md) if this is your first contribution.
- [x] Open an issue or discussion topic to discuss this change.
- [x] Write new tests if applicable.
- [x] Update documentation if applicable.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.7
+    - name: Cache python environment
+      uses: actions/cache@v2
+      with:
+        # Cache pip
+        path: ~/.cache/pip
+        # Check the cache for a given setup.py + MCT version
+        key: ${{ runner.os }}-pip-${{ hashFiles('tfx_addons/version.py') }}
```
Comment (casassg) [inline .github/workflows/ci.yml:44]: should hashFiles here change?

**Review by casassg — APPROVED** [review]
minor issue - happy holidays 😄

File: .github/workflows/ci.yml (hunk 1 above)
Comment (codesue) [reply .github/workflows/ci.yml]: Thanks for catching this!!

**Comment by codesue** [comment]
> minor issue - happy holidays 😄

Happy holidays! 😄

**Pull request merged**